packages/application-shell/src/components/application-shell/application-shell.spec.js

```diff
@@ -817,234 +818,58 @@ describe('when dispatching a loading notification', () => {
   });
 });
 describe('when clicking on navbar menu toggle', () => {
-  it('should expand and collapse menu', async () => {
+  it('should expand menu, show "Support", and collapse menu', async () => {
+    const renderResult = renderApp();
     const { findByLeftNavigation, waitForLeftNavigationToBeLoaded } =
-      renderApp();
+      renderResult;
     await waitForLeftNavigationToBeLoaded();
     const button = await screen.findByTestId('menu-expander');
+
+    // Define variables for container and navbarRendered
+    const container = await findByLeftNavigation();
+    const navbarRendered = within(container);
+
+    // Expand the menu and verify local storage changes
     fireEvent.click(button);
     await waitFor(() => {
       expect(window.localStorage.setItem).toHaveBeenCalledWith(
-        STORAGE_KEYS.IS_FORCED_MENU_OPEN,
-        'false'
+        'MSW_COOKIE_STORE_test',
+        'test'
       );
     });
-    fireEvent.click(button);
+
     await waitFor(() => {
       expect(window.localStorage.setItem).toHaveBeenCalledWith(
-        STORAGE_KEYS.IS_FORCED_MENU_OPEN,
+        'isForcedMenuOpen',
         'true'
       );
     });
-    // Check that the support link is rendered
-    // Get the nav container, to narrow down the search area
-    const container = await findByLeftNavigation();
-    const navbarRendered = within(container);
+
+    // Check that the "Support" link is rendered when the menu is expanded
     expect(navbarRendered.getByText('Support')).toBeInTheDocument();
-  });
-});
-describe('navbar menu links interactions', () => {
```

Comment:
Are these test actually not needed anymore?

Reply:
Hmm, I'm not sure why you think that?
We are checking the links behaviour with this test and I guess we will update it as well when we implement the menu full accessibility.

Reply:
Sorry, I think the diff confused me.

Forget about my comment.